test_collections/tool_blocklist_unit_tests/test_suite_blocklist_1/suite.py

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from app.chip_tool.test_suite import TestSuite
+from test_collections.sdk_tests.support.chip_tool.test_suite import TestSuite
```

Comment:
Why are we import TestSuite from the `chip_tool.test_suite` and not directly from test engine?

```suggestion
from app.test_engine.models import TestSuite
```

Reply:
Or did we mean to use `from test_collections.sdk_tests.support.chip_tool.test_suite import ChipToolSuite` ?

Reply:
Yeah, you're right. Just updated it